README.md

```diff
@@ -106,8 +106,8 @@ A simple Makefile:
 
 TOPLEVEL_LANG = verilog
 VERILOG_SOURCES = $(shell pwd)/dff.sv
-TOPLEVEL = dff
-MODULE = test_dff
+COCOTB_TOPLEVEL = dff
+COCOTB_TEST_MODULES = test_dff
```

Comment:
Thanks! We are keeping the README.md compatible with the current stable release (since people got confused otherwise). So can you please revert this change in the README?

Reply:
Oops done

Reply:
Oh no worries, you couldn't have known. Thanks for the update!